flixel/system/FlxSound.hx

```diff
@@ -442,11 +447,41 @@ class FlxSound extends FlxBasic
 	 */
 	public inline function stop():FlxSound
 	{
-		cleanup(autoDestroy, true, true);
+		cleanup(autoDestroy, true);
 		return this;
 	}
 	
 	/**
+	 * Sets the sound point at which to restart playing after a loop
+	 * @param	newloopPoint The new position in the sound, in milliseconds
+	 * @return	The updated value in milliseconds
+	 */
```

Comment:
There's no point to having documentation for setters, the docs should go to the property.